libqtile/backend/x11/core.py

```diff
@@ -54,8 +54,6 @@
     xcffib.xproto.CreateNotifyEvent,
     xcffib.xproto.FocusInEvent,
     xcffib.xproto.FocusOutEvent,
-    xcffib.xproto.KeyReleaseEvent,
-    # DWM handles this to help "broken focusing windows".
```

Comment:
Is this comment line meant to be deleted with `KeyReleaseEvent` or does it refer to the `MapNotifyEvent` line below? Couldn't really tell from the `git blame`...

Reply:
Eh, remove it, lots of our own MapNotify-related code has grown a bit so that we are handling things our own way.

Reply:
Okay, thanks.